release.sh

```diff
@@ -313,64 +314,105 @@ function build_from_source() {
 }
 
 function get_images_in_yamls() {
-  rm -rf imagerefs.txt
+  rm -rf "$IMAGES_REFS_FILE"
   echo "Assembling a list of image refences to sign"
-  for file in $@; do
+  for file in "$@"; do
     [[ "${file##*.}" != "yaml" ]] && continue
     echo "Inspecting ${file}"
-    for image in $(grep -oh "\S*${KO_DOCKER_REPO}\S*" "${file}"); do
-      echo $image >> imagerefs.txt
-    done
+    while read -r image; do
+      echo "$image" >> "$IMAGES_REFS_FILE"
+    done < <(grep -oh "\S*${KO_DOCKER_REPO}\S*" "${file}")
```

Comment:
Shouldn't block the merge, but I prefer and find it easier to read the `for x in blah` loop over the `while do done < input` syntax.

Reply:
https://www.shellcheck.net/wiki/SC2013

Shell is hard.

Reply:
Yes it is :(

Shellcheck folks are more bash nerds than me for sure so I will mark as resolved

Reply:
BTW: We should have shellcheck linter running before merge on this repo.

Reply:
100% https://github.com/knative/hack/issues/269